### PR TITLE
[CDAP-6371] Add Joiner widget json

### PR DESCRIPTION
**Pull request by elfenheart**

### CODE REVIEW

File: core-plugins/widgets/Joiner-batchjoiner.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "metadata": {
+    "spec-version": "1.3"
+  },
+  "inputs": {
+    "multipleInputs": true
```
Comment (CuriousVini) [inline core-plugins/widgets/Joiner-batchjoiner.json:6]: Does this property mean we only allow more than 2 inputs for joiner plugin and not any other plugin?

**Comment by CuriousVini** [comment]
Just one question. Otherwise LGTM

File: core-plugins/widgets/Joiner-batchjoiner.json (hunk 1 above)
Comment (elfenheart) [inline core-plugins/widgets/Joiner-batchjoiner.json:6]: This property means that the plugin allow multiple inputs with different schema. If the config does not have this property, it means that the schema for all inputs have to be the same

**Comment by ajainarayanan** [comment]
👍 LGTM

**Pull request merged**